test/python/importer/jit_ir/ivalue_import/tensors.py

```diff
@@ -55,10 +55,14 @@ def __init__(self):
 # CHECK:   torch.slot "ones_qint8", %[[ONES_QINT8]] : !torch.tensor<[1],!torch.qint8>
 # CHECK:   torch.slot "ones_quint8", %[[ONES_QUINT8]] : !torch.tensor<[1],!torch.quint8>
 # CHECK: }
-
-
 test_module = TestModule()
 recursivescriptmodule = torch.jit.script(test_module)
+
+import_options = ImportOptions()
+import_options.assumeTensorsHaveValueSemantics = False
```

Comment:
can we make this the default (i.e. retain the old behavior) so that we don't need to add custom import options everywhere?

Reply:
done.